docs/migration/dapp_migration.md

```diff
@@ -0,0 +1,46 @@
+# Nitro Migration Notes for Solidity Devs: Living Document
+
+
+The following is a summary of changes in the upcoming Arbitrum One chain's Nitro upgrade that dapp devs should be aware of. 
+
+It reflects the current state of Nitro and should be considered a living document; things might change before launch.
+
+_Last Updated 4/13/2022_
+
+## Cool New Stuff 
+
+For starters, here's a sampling of exciting perks dapps with get with the Nitro upgrade:
+
+- **Ethereum L1 Gas Compatibility 🥳**:  gas pricing and accounting for EVM operations is be perfectly in line with L1; no more ArbGas/Eth Gas conversion.  
+- **Safer Retryable tickets 🥳**: Retryable tickets' submission cost is collected in the L1 Inbox contract; if the submission cost is too low, the transaction will simply revert on the L1 side, eliminating the [failure mode](https://developer.offchainlabs.com/docs/l1_l2_messages#important-note-about-base-submission-fee) in which a retryable ticket fails to get created. 
+- **Calldata compression 🥳**: Compression takes place protocol level; dapps don't need to change anything, data will just get cheaper! (You are charged even less if your calldata is highly compressible with brotli.)
+- **Support for All Ethereum L1 precompiles 🥳**: (`blake2f`, `ripemd160`, etc)
+- **Tighter Syncronization with L1 Block Numbers / Timestamps 🥳**:  L1 block number and timestamps (accessed via `block.number` and `block.timestamp` on L2) are updated more frequently in Nitro than in Arbitrum classic; expect them to be nearly real-time/ in sync with L1. 
+
+- **L2 Block hash EVM Consistency 🥳**: L2 block hashes take the same format as on Ethereum (if you query it from the ArbSys precompile, not the one in `block.hash(uin256)`).
+
+
+- **Geth tracing 🥳**: `debug_traceTransaction` RPC endpoint is supported; this includes tracing of ArbOS internal bookkeeping actions.
+
+## Breaking changes
```

Comment:
Hmm would it make sense to maybe add a section about RPC?
Some custom fields have changed (like instead of `feeStats` we have `gasUsed` and `gasUsedForL1` which give functionally the same data)
There's also no more `l1SequenceNumber` that some projects used to use to find out if a tx was included in a L1 batch. With nitro you can now just query a node-interface precompile https://github.com/OffchainLabs/nitro/pull/505
And yea, the tracing stuff as well